cap-notebook/version.txt

```diff
@@ -1 +1 @@
-1.0.3
+1.0.2
```

Comment:
Shouldn't be changed, looks like changed by accident.

Reply:
stimmt, gehe ich nochmal durch bevor ich ihn auf Review setze, danke für die Info